### DISCUSSION TITLE
Forest 🌲 feature requests

### DISCUSSION THREAD

**Discussion by zettai-caiman** [ChainSafe/forest#2265] [Ideas]
A place for external contributors, users, and teams to request and discuss Forest 🌲 feature sets.

**Comment by zettai-caiman** [comment]
Running lotus-miner on Forest. The ability for node operators to switch between Lotus and Forest with ease - suggested by @jennijuju  from Protocol Labs.

**Comment by lemmih** [comment]
A light-client in the style of Helios: https://twitter.com/NoahCitron/status/1589739389545914368?s=20&t=i66Q5jrdr7toyaEogjg4YQ

It uses centralized servers but has a system for verifying their correctness without having to trust them.
